search_params.py

```diff
@@ -91,17 +99,25 @@ def init_search_algorithm(search_alg, metric=None, mode=None):
     logging.info(f'{search_alg} search is found, run BasicVariantGenerator().')
 
 
+def load_static_data(config):
+    datasets = data_utils.load_datasets(config)
+    return {
+        "datasets": datasets,
+        "word_dict": data_utils.load_or_build_text_dict(config, datasets['train']),
+        "classes": data_utils.load_or_build_label(config, datasets)
+    }
+
+
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument(
         '--config', help='Path to configuration file (default: %(default)s). Please specify a config with all arguments in LibMultiLabel/main.py::get_config.')
     parser.add_argument('--cpu_count', type=int, default=4, help='Number of CPU per trial (default: %(default)s)')
     parser.add_argument('--gpu_count', type=int, default=1, help='Number of GPU per trial (default: %(default)s)')
     parser.add_argument('--local_dir', default=os.getcwd(), help='Directory to save training results of tune (default: %(default)s)')
-    parser.add_argument('--num_samples', type=int, default=50, help='Number of running samples (default: %(default)s)')
+    parser.add_argument('--num_samples', type=int, default=1, help='Number of running samples (default: %(default)s)')
```

Comment:
Should we update rcv1/cnn_tune.yml and caml/caml_tune.yml if we modified the default value here?

Reply:
Sure. I think we can also add a message to the user that they should set `num_samples` to 1 if they're running grid search.